docs/guides/ORCHESTRATION_CHAT_COMPLETION.md

```diff
@@ -259,6 +259,39 @@ try (Stream<String> stream = client.streamChatCompletion(prompt, config)) {
 Please find [an example in our Spring Boot application](../../sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/services/OrchestrationService.java).
 It shows the usage of Spring Boot's `ResponseBodyEmitter` to stream the chat completion delta messages to the frontend in real-time.
 
+
+## Add images and multiple text inputs to a message
+
+It's possible to add images and multiple text inputs to a message.
+
+### Add images to a message
+
+There are multiple ways to add an image to a user message as detailed below. Note, that currently only user messages are supported for image attachments.
+
+```java
+var message1 = Message.user("What is in this image?").addImage("https://url.to/image.jpg");
+var message2 = Message.user("And what is in this image?").add(MessageContent.image("https://url.to/image.jpg"));
```

Comment:
(minor) Users may wonder why we have two examples here, and if they are different in any way. Personally, I'd KISS and only give one example.

What is not obvious here is if a message can contain only an image, without any text